src/tools/CMakeLists.txt

```diff
@@ -46,9 +48,15 @@ if(WITH_LIBCEPHFS)
   install(TARGETS ceph-client-debug DESTINATION bin)
 endif(WITH_LIBCEPHFS)
 
+if(WITH_TESTS)
```

Comment:
@rjfd you might want to move this block into https://github.com/ceph/ceph/pull/10872/files#diff-65c9cee53e3f940aed5b8ad9942ff108L44 .